client/src/main/java/org/spine3/base/Stringifiers.java

```diff
@@ -154,9 +158,79 @@ private Stringifiers() {
     }
 
     /**
+     * Obtains {@code Stringifier} for the string values.
+     *
+     * @return the stringifier for the string values
+     */
+    static Stringifier<String> stringStringifier() {
+        final Stringifier<String> stringStringifier = new StringStringifier();
+        return stringStringifier;
+    }
+
+    /**
+     * Obtains {@code Stringifier} for list with default delimiter for the passed list elements.
+     *
+     * @param elementClass the class of the list elements
+     * @param <T>          the type of the elements in this list
+     * @return the stringifier for the list
+     */
+    public static <T> Stringifier<List<T>> listStringifier(Class<T> elementClass) {
+        checkNotNull(elementClass);
+        final Stringifier<List<T>> listStringifier = new ListStringifier<>(elementClass);
+        return listStringifier;
+    }
+
+    /**
+     * Obtains {@code Stringifier} for list with custom delimiter for the passed list elements.
+     *
+     * @param elementClass the class of the list elements
+     * @param delimiter    the delimiter or the list elements passed via string
+     * @param <T>          the type of the elements in this list
+     * @return the stringifier for the list
+     */
+    public static <T> Stringifier<List<T>> listStringifier(Class<T> elementClass,
+                                                           char delimiter) {
+        checkNotNull(elementClass);
+        checkNotNull(delimiter);
+        final Stringifier<List<T>> listStringifier =
+                new ListStringifier<>(elementClass, delimiter);
+        return listStringifier;
+    }
+
+    /**
+     * Create the {@code Escaper} which contains will escape '\' and passed character.
+     *
+     * @param charToEscape the char to escape
+     * @return the constructed escaper
+     */
+    static Escaper createEscaper(char charToEscape) {
+        final String escapedChar = "\\" + charToEscape;
+        final Escaper result = Escapers.builder()
+                                       .addEscape('\"', "\\\"")
+                                       .addEscape(charToEscape, escapedChar)
+                                       .build();
+        return result;
+    }
+
+    /**
+     * The {@code Stringifier} for the {@code String} values.
+     */
+    private static class StringStringifier extends Stringifier<String> {
```

Comment:
That probably should be `NoopStringifier` which description says it does nothing.

Same with the static `Stringifiers.stringStringifier()` method.